.github/workflows/e2e_on_pull.yaml

```diff
@@ -0,0 +1,164 @@
+# (C) Copyright Confidential Containers Contributors 2023.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Run end-to-end (e2e) tests on pull request.
+---
+name: e2e tests
+
+on:
+  pull_request_target:
+    types:
+      # This workflow will be run if the pull request is labeled test_e2e_libvirt, so
+      # adding 'labeled' to the list of activity types.
+      #
+      - opened
+      - synchronize
+      - reopened
+      - labeled
+    branches:
+      - 'main'
+
+env:
+  # cloud-api-adaptor image registry
+  E2E_IMG_REGISTRY: ghcr.io/${{ github.repository_owner }}
+  # cloud-api-adaptor: image release tag
+  E2E_IMG_RELEASE_TAG: ci-pr${{ github.event.number  }}
+  # cloud-api-adaptor image dev tag
+  E2E_IMG_DEV_TAG: ci-pr${{ github.event.number  }}-dev
+
+jobs:
+  authorize:
+    runs-on: ubuntu-latest
+    if: ${{ contains(github.event.pull_request.labels.*.name, 'test_e2e_libvirt') }}
+    steps:
+      - run: true
```

Comment:
it's a dummy, but it should be `- run: "true"` otherwise it's interpreted as boolean

Reply:
hey @mkulke, first of all, thanks for the review. For this comment in particular, yeah, I hadn't tested that code path...good catch. Fixed.